keras_nlp/src/models/video_swin/video_swin_layers_test.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras import ops
+
+from keras_nlp.src.models.video_swin.video_swin_layers import (
+    VideoSwinPatchingAndEmbedding,
+)
+from keras_nlp.src.models.video_swin.video_swin_layers import (
+    VideoSwinPatchMerging,
+)
+from keras_nlp.src.models.video_swin.video_swin_layers import (
+    VideoSwinWindowAttention,
+)
+from keras_nlp.src.tests.test_case import TestCase
+
+
+class TestVideoSwinPatchingAndEmbedding(TestCase):
+    def test_patch_embedding_compute_output_shape(self):
+        patch_embedding_model = VideoSwinPatchingAndEmbedding(
+            patch_size=(2, 4, 4), embed_dim=96, norm_layer=None
+        )
+        input_array = ops.ones(shape=(1, 16, 32, 32, 3))
+        output_shape = patch_embedding_model(input_array).shape
+        expected_output_shape = (1, 8, 8, 8, 96)
+        self.assertEqual(output_shape, expected_output_shape)
+
+    def test_patch_embedding_get_config(self):
+        patch_embedding_model = VideoSwinPatchingAndEmbedding(
+            patch_size=(4, 4, 4), embed_dim=96
+        )
+        config = patch_embedding_model.get_config()
+        assert isinstance(config, dict)
```

Comment:
rewrite all of these tests to match the repo style. We do not use `assert` anywhere.